airflow/api/common/experimental/pool.py

```diff
@@ -51,6 +51,11 @@ def create_pool(name, slots, description, session=None):
     except ValueError:
         raise AirflowBadRequest("Bad value for `slots`: %s" % slots)
 
+    # Get the length of the pool column
+    pool_name_length = Pool.pool.property.columns[0].type.length
+    if len(name) > pool_name_length:
+        raise AirflowBadRequest("Pool name shouldn't be more than %s characters" % pool_name_length)
```

Comment:
```suggestion
        raise AirflowBadRequest("Pool name can't be more than %d characters" % pool_name_length)
```

Reply:
Thanks for this review. I'll make the change now